docker/pypi/wmagent/run.sh

```diff
@@ -191,209 +115,141 @@ deploy_to_host(){
     echo "-------------------------------------------------------"
     echo "Start: $stepMsg"
 
-    # Check if the host has all needed components' logs and job cache areas
-    # TODO: purge job cache if -p run option has been set
-    echo "$FUNCNAME: Initialise install"
-    _init_valid $WMA_INSTALL_DIR || {
-        mkdir -p $WMA_INSTALL_DIR/{wmagent,mysql,couchdb} && echo $WMA_BUILD_ID > $WMA_INSTALL_DIR/.dockerInit
-        [[ -h $WMA_INSTALL_DIR/wmagentpy3 ]] || ln -s $WMA_INSTALL_DIR/wmagent $WMA_INSTALL_DIR/wmagentpy3
-    }
-
-    # Check if the host has all config files and copy them if missing
-    # NOTE: The final config/.dockerInit is to be set after full agent initialisation during `agent_upload_config`
-    echo "$FUNCNAME: Initialise config"
-    local serviceList="wmagent mysql couchdb rucio"
-    local config_mysql=my.cnf
-    local config_couchdb=local.ini
-    local config_wmagent=manage
-    local config_rucio=rucio.cfg
-    for service in $serviceList; do
-        _init_valid $WMA_CONFIG_DIR/$service && continue
-        echo "$FUNCNAME: config service=$service"
-        local errVal=0
-        local config=config_$service && config=${!config}        # expanding to the proper config name
-        if [[ $service = "wmagent" ]]
-        then
-            [[ -d $WMA_CONFIG_DIR/$service ]] || mkdir -p $WMA_CONFIG_DIR/$service ; let errVal+=$?
-            cp -f $WMA_DEPLOY_DIR/bin/${config} $WMA_CONFIG_DIR/$service/ ; let errVal+=$?
-            chmod 755 $WMA_CONFIG_DIR/$service/$config;
-            [[ -h $WMA_CONFIG_DIR/wmagentpy3 ]] || ln -s $WMA_CONFIG_DIR/$service $WMA_CONFIG_DIR/wmagentpy3
-        elif [[ $service = "rucio" ]]
-        then
-            [[ -d $WMA_CONFIG_DIR/$service/etc ]] || mkdir -p $WMA_CONFIG_DIR/$service/etc ; let errVal+=$?
-            cp -f $WMA_DEPLOY_DIR/etc/${config} $WMA_CONFIG_DIR/$service/etc/ ; let errVal+=$?
-        else
-            [[ -d $WMA_CONFIG_DIR/$service ]] || mkdir -p $WMA_CONFIG_DIR/$service ; let errVal+=$?
-            cp -f $WMA_DEPLOY_DIR/etc/${config} $WMA_CONFIG_DIR/$service/ ; let errVal+=$?
-        fi
-        [[ $errVal -eq 0 ]] && echo $WMA_BUILD_ID > $WMA_CONFIG_DIR/$service/.dockerInit
-    done
+    # TODO: This is to be removed once we decide to run it directly from the deploy area
```

Comment:
@amaltaro I'd like to draw your attention to these two lines here. To me these are obsolete. I kept them just for legacy reasons, but the way how things are currently organized, there is no need to copy the manage script  from the $WMA_DEPLOY_DIR to $WMA_MANAGE_DIR both are listed in the agent's PATH, so the script would be easily run from any of those locations. The only difference is that if we use it from $WMA_DEPLOY_DUR it would be run from the docker image, and would never show up in the host mount area ($WMA_MANAGE_DIR is equivalent currently to $WMA_CONFIG_DIR).  The later gives  us a little bit of freedom - we will have the ability to test some new changes or fix bugs to the `manage` script without the need to build a new container every time. And this was the only reason I have not removed this step at once. I'd like to hear your opinion on this one as well.